app/views/custom_attributes/string_values/_show_string_value.html.erb

```diff
@@ -1,6 +1,10 @@
 <% cssStyles ||= {} %>
```

Comment:
I think this is an existing bug and can be fixed in this PR. cssStyles is not used anywhere. I believe the developer meant `cssClasses` here. So can you change `cssStyles` to `cssClasses`?